src/components/menu/styledMenu.js

```diff
@@ -0,0 +1,56 @@
+import styled from 'styled-components';
+import { WorkWithUs } from '../sharedComponents/sharedComponents';
+
+export const Wrapper = styled.div`
+    width: 100%;
+    max-width: 375px;
+    display: flex;
+    flex-direction: column;
+    position: absolute;
+    top: 60px;
+    right: 20%;
```

Comment:
i think we should change this to 80 so it's right below the header

Reply:
When I chose this, I was doing it based on the smaller screen width header heights (which are 60) but it exists at higher screen widths now... so good call